src/apps/popup/app-router.tsx

```diff
@@ -256,7 +256,7 @@ function AppRoutes() {
         path={RouterPath.DownloadedSecretKeys}
         element={<DownloadedSecretKeysPage />}
       />
-      <Route path={RouterPath.Transfer} element={<TransferPage />} />
+      <Route path={RouterPath.TransferERC20} element={<TransferPage />} />
```

Comment:
I guess we need to rename this route path. It is responsible for native transfer too

Reply:
how do you want to rename it?

Reply:
I think `TokenTransfer` or `Transfer`.
Now this route is responsible for erc20 and native transfer, so it's confusing when it names `TransferERC20`